xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2358,24 +2358,24 @@ xe.admin.passwordReset.step2.title=Reset your password
 xe.admin.passwordReset.step2.newPassword.label=New password
 xe.admin.passwordReset.step2.newPasswordVerification.label=Re-enter new passwod
 xe.admin.passwordReset.step2.submit=Save
-xe.admin.passwordReset.step2.success=The password has been successfully set.
+xe.admin.passwordReset.step2.success=The password has been successfully reset.
 xe.admin.passwordReset.step2.login=Please login to continue \u00BB
 xe.admin.passwordReset.step2.backToStep1=Back to the password reset page \u00BB
 xe.admin.passwordReset.step2.error.emptyPassword=The password cannot be empty.
 xe.admin.passwordReset.step2.error.verificationMismatch=The two passwords do not match.
 xe.admin.passwordReset.step2.error.wrongParameters=Wrong parameters.
-xe.admin.passwordReset.step2.error.noProgrammingRights=This page requires programming rights to work, which currently isn't the case. Please notify an administrator of this problem and try again later.
+xe.admin.passwordReset.step2.error.noProgrammingRights=This page requires programming rights to work, which currently isn't the case. Please notify an administrator for this problem and try again later.
```

Comment:
I don't understand why you changed it here, `of` sounded correct to me (but I might be wrong).

Reply:
I think that the user should be notifying the administrator for the concerned problem

Reply:
in my opinion

Reply:
As far as I know, the rule for business English is to use "notify of" (at least, I didn't found any references to "notify for" [here](https://dictionary.cambridge.org/dictionary/english/notify)) ; so we better keep the current translation :) .

Reply:
+1 to keep the current translation